schema/snapcraft.json

```diff
@@ -889,8 +890,7 @@
                         "type": {
                             "enum": [
                                 "app",
-                                "gadget",
-                                "kernel"
+                                "gadget"
```

Comment:
This no longer matches usage on line 887 of the context.

Reply:
thanks @zyga